src/certificate.ts

```diff
@@ -207,7 +212,11 @@ function renderFirstPage(
   });
 }
 
-function renderSecondPage(doc: PDFKit.PDFDocument, topics: CourseTopics) {
+function renderSecondPage(
+  doc: PDFKit.PDFDocument,
+  topics: CourseTopics | DataCourseTopics,
```

Comment:
Maybe add a new type called `Topics`.

```suggestion
  topics: Topics,
```